package.json

```diff
@@ -119,7 +121,14 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run test:changed"
+      "pre-commit": "lint-staged"
     }
+  },
+  "lint-staged": {
+    "src/**/*.{tsx,ts}": [
+      "npm run test:staged",
```

Comment:
Maybe we could use yarn here as well? I don't know does it make any different or is it possible, but it would be nice to be consistent

Reply:
yeah maybe yarn is better for consitency